README.rst

```diff
@@ -1,3 +1,5 @@
+**DEPRECATED** This formula is not maintained, you should better use https://github.com/saltstack-formulas/hostsfile-formula
```

Comment:
@daks Do you mind doing this the same way as https://github.com/saltstack-formulas/salt-virt-formula?  We can then archive the formula at the same time.

Reply:
something like that? (see update)

Reply:
@daks Not sure why but I don't see the update.  In any case, it's not that important so let's move forward this.  Thanks!

Reply:
@daks I realise what happened!  The original PR was in a `deprecation` branch in _this_ repo and your update was in the `deprecation` branch in _your_ repo!

* https://github.com/daks/hosts-formula/tree/deprecation

I've had this problem in the past as well.  If you want to use that branch as another PR, we can finalise everything here.

Reply:
@daks Used your branch in #4 and the formula has been archived.